src/types/index.js

```diff
@@ -0,0 +1,22 @@
+// Centralised type definitions for commonly used ones
```

Comment:
Maybe we should divide types for specific components. Example: src/types/item.js

Reply:
Yes, if we will add more in future it is definitely good idea.